eventually/authentication/models.py

```diff
@@ -60,10 +88,14 @@ def delete_by_id(user_id):
             user.delete()
             return True
         except CustomUser.DoesNotExist:
-            pass
+            return -1
 
     @staticmethod
-    def create(first_name, middle_name, last_name, email, password):
+    def create(first_name='first-name',
```

Comment:
is it a good idea to set the default values for this method fields? If I send the only name it means that my last name would be like default last name or not?

Reply:
Makes sense. Fixed.